core/lpms.go

```diff
@@ -131,6 +135,20 @@ func (l *LPMS) HandleHLSPlay(
 	l.vidPlayer.HandleHLSPlay(getMasterPlaylist, getMediaPlaylist, getSegment)
 }
 
+func (l *LPMS) RTMPToHLS(s *segmenter.FFMpegVideoSegmenter, ctx context.Context, cleanup bool) error{
```

Comment:
Since this doesn't use any instance members and is internal so far, maybe we can change the function to be static and/or even private. Then to make testing easier, pass in the `segmenter.VideoSegmenter` interface, eg `func rtmpToHLS(ctx context.Context, s segmenter.VideoSegmenter, cleanup bool)`

See the note down below about testing changes.

Reply:
Done